argschema/fields/image.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
```

Comment:
Yeah, that is a minor inconvenience but looking forward to Python 3.  The other problem in either version is that we have to patch in jpeg2000 and bigtiff support.  This is supported by by pathlib, but I have to figure out what the magic numbers are for those new formats.